R/teal_slice.R

```diff
@@ -481,50 +520,58 @@ c.teal_slices <- function(...) {
   )
 }
 
+#' @rdname teal_slice
+#' @keywords internal
+#'
+slices_to_list <- function(tss) {
+  slices_list <- lapply(tss, as.list)
+  attrs <- attributes(unclass(tss))
+  tss_list <- list(slices = slices_list, attributes = attrs)
+  Filter(Negate(is.null), tss_list) # drop attributes if empty
+}
 
-# format method for teal_slices
+#' @param show_all `logical(1)` should parameters set to NULL be returned
```

Comment:
`show_all` is documented with other arguments at the top of the file

Reply:
e4fdff5046ef1b15fb1ffe9c71caab9e728af23a